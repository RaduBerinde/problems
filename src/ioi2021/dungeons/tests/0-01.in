b50747e9-747c-4fca-b3b0-62317b32d2f6
3 2
2 6 9
3 1 2
2 2 3
1 0 1
0 1
2 3
