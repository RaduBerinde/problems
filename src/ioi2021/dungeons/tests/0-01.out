f39eb8f7-7d10-4b4a-af02-d7aef3d4dd0a
OK
24
25
